grype/presenter/table/presenter.go

```diff
@@ -55,7 +55,7 @@ func (pres *Presenter) Present(output io.Writer) error {
 			fixVersion = ""
 		}
 
-		rows = append(rows, []string{m.Package.Name, m.Package.Version, fixVersion, m.Vulnerability.ID, severity})
+		rows = append(rows, []string{m.Package.Name, m.Package.Type, m.Package.Version, fixVersion, m.Vulnerability.ID, severity})
```

Comment:
Note: `m.Package.Type` will have to be converted to a `string` type for this to compile successfully.

Reply:
Ah yes, sorry not to have tried it locally already - I've hit some issues during dev setup and (foolishly!) thought I might get away with a nice simple change... but have now resolved these, got a working local setup & pushed a fix for this 👍